deploy/runtools/firesim_topology_elements.py

```diff
@@ -325,18 +326,22 @@ def copy_back_job_results_from_run(self, slotno: int) -> None:
             rootLogger.debug("[localhost] " + str(localcap))
             rootLogger.debug("[localhost] " + str(localcap.stderr))
 
+        dest_sim_dir = self.get_host_instance().get_sim_dir()
+
         # mount rootfs, copy files from it back to local system
         rfsname = self.get_rootfs_name()
         if rfsname is not None:
             is_qcow2 = rfsname.endswith(".qcow2")
-            mountpoint = """/home/centos/sim_slot_{}/mountpoint""".format(simserverindex)
+            mountpoint = """{}/sim_slot_{}/mountpoint""".format(dest_sim_dir, simserverindex)
             with StreamLogger('stdout'), StreamLogger('stderr'):
                 run("""sudo mkdir -p {}""".format(mountpoint))
 
                 if is_qcow2:
-                    rfsname = self.get_host_instance().nbd_tracker.get_nbd_for_imagename(rfsname)
+                    host_inst = self.get_host_instance()
+                    assert isinstance(host_inst.instance_deploy_manager, EC2InstanceDeployManager)
```

Comment:
Ideally yes, but for now we only support the NBD stuff on AWS EC2 (other deploy managers will not have this by default). Therefore, this has to be specific to the EC2InstanceDeployManager.

Reply:
I'd prefer if the DeployManager explicitly called out if it supported QCOW images, (e.g., it could have a member indicating that), or we just rip out the unused qcow support altogether.

Reply:
Or if its better, I can do this in a follow up PR so that this PR gets in an unblocks Sagars meta-sim PR.

Reply:
You can do it in a separate PR.